walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -31,22 +31,39 @@
       return spots;
```

Comment:
how about just ```Arrays.asLIst()```? which is more straightforward.
And you could declare the spots inline below
```Collection<TimeRange> spots = findMeetingsSpots(...) ```